Thyme/Thyme.js

```
@@ -0,0 +1,311 @@
+
+        // Then check custom codecs
+        const codec = this.#codecs.get(type);
+        if (!codec) {
+            throw new Error(`No codec registered for type ${type}`);
+        }
+        return codec;
+    }
+
+    static write(type, buffer, posRef, data) {
+        if (buffer.every(Number.isInteger)) {
+            buffer = new Uint8Array(buffer).buffer;
```

Biome

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)